### PR TITLE
Substituting the winston package by the logops one

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% test passed

- Fixes https://github.com/telefonicaid/fiware-sth-comet/issues/187
- Makes not applicable: https://github.com/telefonicaid/fiware-sth-comet/issues/11

### CODE REVIEW

File: lib/sth.js [hunk 1]
```diff
@@ -84,31 +90,34 @@ function exitGracefully(err, callback) {
...
   }
 
   var version = sthHelper.getVersion();
   sthLogger.info(
-    'Starting up STH server version %s...',
-    version.version,
     {
-      operationType: sthConfig.OPERATION_TYPE.SERVER_START
-    }
+      op: sthConfig.OPERATION_TYPE.SERVER_START
```
Comment (dmoranj) [inline lib/sth.js]: You are repeating these objects over and over again. Better declare them as variables at the top of the page to improve readability of the rest of the file.

Comment (gtorodelvalle) [inline lib/sth.js]: Agreed... ;) On it... :)

Comment (gtorodelvalle) [inline lib/sth.js]: Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/1f9ae996f0f354070de2ec9e8325ccb3831a1326

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**